src/coreclr/nativeaot/System.Private.DisabledReflection/src/Internal/Runtime/CompilerHelpers/LibraryInitializer.cs

```diff
@@ -1,17 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using Internal.Reflection;
-using Internal.Reflection.Augments;
-using Internal.Runtime.Augments;
-
-namespace Internal.Runtime.CompilerHelpers
-{
-    public class LibraryInitializer
-    {
-        public static void InitializeLibrary()
-        {
-            ReflectionAugments.Initialize(new ReflectionCoreCallbacksImplementation());
```

Comment:
We can also delete `ReflectionCoreCallbacks` abstraction - move the implementation to ReflectionAugments and make the methods static (it is fine to do it as a follow up).

Reply:
Sounds good!